plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1387,6 +1388,12 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
       val fromX = focalPoint.x.toDouble()
       val fromY = focalPoint.y.toDouble()
 
+      // Skip invalid focal points with non-finite values
+      if (!focalPoint.x.isFinite() || !focalPoint.y.isFinite()) {
```

Comment:
looks good for a temporary fix, but we should also investigate the root cause, seems the focal point is coming from the MultiFingerGesture class in MapboxGestures library, would be good to fix it there, as the focal point is also used in several other places in the gestures plugin.

Reply:
we don't have reproducible use-case sadly so let's fix precisely an issue for now and indeed take a deeper look later if needed